app/code/Magento/InventoryCatalog/Setup/InstallData.php

```diff
@@ -54,27 +55,35 @@ class InstallData implements InstallDataInterface
     private $assignSourcesToStock;
 
     /**
+     * @var StockItemIndexerInterface
+     */
+    private $stockItemIndexer;
+
+    /**
      * @param SourceRepositoryInterface $sourceRepository
      * @param SourceInterfaceFactory $sourceFactory
      * @param StockRepositoryInterface $stockRepository
      * @param StockInterfaceFactory $stockFactory
      * @param AssignSourcesToStockInterface $assignSourcesToStock
      * @param DataObjectHelper $dataObjectHelper
+     * @param StockItemIndexerInterface $stockItemIndexer
      */
     public function __construct(
         SourceRepositoryInterface $sourceRepository,
         SourceInterfaceFactory $sourceFactory,
         StockRepositoryInterface $stockRepository,
         StockInterfaceFactory $stockFactory,
         AssignSourcesToStockInterface $assignSourcesToStock,
-        DataObjectHelper $dataObjectHelper
+        DataObjectHelper $dataObjectHelper,
+        StockItemIndexerInterface $stockItemIndexer
     ) {
         $this->sourceRepository = $sourceRepository;
         $this->sourceFactory = $sourceFactory;
         $this->stockRepository = $stockRepository;
         $this->stockFactory = $stockFactory;
         $this->assignSourcesToStock = $assignSourcesToStock;
         $this->dataObjectHelper = $dataObjectHelper;
+        $this->stockItemIndexer = $stockItemIndexer;
```

Comment:
could you please fix naming of this function

```
    /**
     * Assign default stock to default source
     *
     * @return void
     */
    private function assignStockToSource()
    {
        $this->assignSourcesToStock->execute([1], 1);
    }
```

Because we assign Sources to Stock , not vice versa but function name is `assignStockToSource`